Jellyfin.Plugin.KodiSyncQueue/EntryPoints/UserSyncNotification.cs

```diff
@@ -43,9 +44,9 @@ public Task RunAsync()
             return Task.CompletedTask;
         }
 
-        private bool FilterItem(BaseItem item, out int type)
+        private bool FilterItem(BaseItem item, out MediaType type)
```

Comment:
this seems to be a more-or-less duplicate of `LibrarySyncNotification.FilterItem`, maybe extract common part somewhere?